src/main/java/org/fogbowcloud/arrebol/core/models/task/Task.java

```diff
@@ -78,19 +54,20 @@ public void setState(TaskState newState) {
         this.state = newState;
     }
 
-    public List<Command> getAllCommands() {
+    public List<Command> getCommands() {
         return commands;
     }
 
-    public String getId() {
-        return this.id;
+    public void putMetadata(String attributeName, String value) {
```

Comment:
why do we need this? It seems the metadata should be given at creation time, via the constructor

Reply:
We do not need. It was removed and passed to constructor.